### ISSUE TITLE
BUG: Wacom Tablet drawn in Incorrect Position

### ISSUE THREAD

**Issue by CrystalMV13** [zsviczian/obsidian-excalidraw-plugin#2013]
### Have you searched for existing issues (including closed ones)?

- [X] Yes, I have searched the existing issues.

### Does this bug persist in a new vault with only Excalidraw installed?

- [X] Yes, I have verified the issue persists.

### Your environment

SYSTEM INFO:
	Obsidian version: v1.6.7
	Installer version: v1.6.7
	Operating system: #1 SMP PREEMPT_DYNAMIC Fri Aug 30 04:50:21 UTC 2024 (54e6893) 6.10.7-1-default
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: dark
	Community theme: Minimal v7.5.2
	Snippets enabled: 9
	Restricted mode: off
	Plugins installed: 18
	Plugins enabled: 13
		1: Style Settings v1.0.8
		2: Slide Note v0.0.16
		3: Quoth v0.7.1
		4: Outliner v4.8.1
		5: Minimal Theme Settings v7.5.0
		6: Metatable v0.14.4
		7: Latex Suite v1.9.2
		8: Hover Editor v0.11.17
		9: Excalidraw v2.1.6
		10: Desmos v0.6.6
		11: Remotely Sync v0.4.48
		12: Better Math in Callouts & Blockquotes v0.3.7
		13: Callout Manager v1.1.0

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

### Describe the bug

I am using OpenSUSE Tumbleweed, and I have only noticed this bug recently; last time I used Excalidraw, it was on Windows, and the issue was not present.

When I try to draw with my Wacom Tablet, and the crosshair cursor appears, it is drawn to the left and up from where it should be. I.E., buttons highlight as if the "actual cursor" is a bit down and to the right from the "seen cursor", and it is extremely annoying to draw, as I have to estimate where the "actual cursor" is.

Additionally, when I minimize the window, the cursor seems to snap to its actual position once Obsidian is minimized. No other app seems to have this issue, so it seems to have to do with either Obsidian or the Excalidraw plugin.

### Steps to reproduce

1. Open an excalidraw note
2. With a Wacom tablet, or perhaps any drawing tablet, get the crosshair to appear (not sure under what circumstances exactly it appers? Occasionally the mouse cursor is used when in pen mode, but only the crosshair has the misplacement issue)
3. Try to draw in the note, or press any button on the window (even outside of Excalidraw) using the drawing tablet

### Expected behavior

The crosshair icon should align with where the cursor is actually pointing to.

### Additional context

[Obsidian Cursor Bug.webm](https://github.com/user-attachments/assets/43e498e2-64be-48f1-98f7-9cd1d55413b4)

**Comment by zsviczian** [comment]
Please search closed issues a bit more. I see you ticked the box that you have searched existing issue... however I distinctly remember that this has surfaced a number of times. The solution is an update or fix of the linux wacom driver.

**Comment by CrystalMV13** [comment]
I'm sorry, I did search the closed issues as well, and I don't see any of this sort, only ones about the cursor not being visible at all. I only see one issue now that is my issue -- #1755, and it is open. I will look into updating the driver.

**Comment by zsviczian** [comment]
ah... sorry for that. regardless, unfortunately I can't really help. What I know, is that positioning on my Huion drawing tablet, Samsung S23 Ultra, iPad + Pencil, and windows 11 laptop with stylus is good.

You can try to install Excalidraw in an empty/new Vault with default settings to see if the issue still persists.

**Comment by CrystalMV13** [comment]
Yes, I did do that, and the issue still persisted. If there is some way to not bring up the crosshair, and only use the default cursor for pen mode, I would be happy with that, but I don't see any such setting.

I also tested on excalidraw.com (on a Chromium-based browser) and the issue does not persist there. It seems to only happen with this plugin on Obsidian, though it does effect the whole of Obsidian. My drivers all seem to be up to date as well, and everything seems fine everywhere else so I hesitate to say they're broken. I guess I'll have to troubleshoot more.
